### PR TITLE
Customer specific pricing

### PR DESCRIPTION
**Pull request by paales**

### CODE REVIEW

File: packages/magento-product/components/AddProductsToCart/AddProductsToCartFab.tsx [hunk 1]
```diff
@@ -22,7 +22,7 @@ export function AddProductsToCartFab(props: AddProductsToCartFabProps) {
       {...rest}
       {...action}
       icon={showSuccess && !action.loading ? iconCheckmark : icon}
-      aria-label={i18n._(/* i18n*/ `Add to Cart`)}
+      aria-label={t`Add to Cart`}
```
[the remark this answers is not in the document]
Comment (paales) [reply packages/magento-product/components/AddProductsToCart/AddProductsToCartFab.tsx:25]: Nope, aria-label expects a string, not a react component

File: packages/magento-product/tsconfig.json [hunk 2]
```diff
@@ -1,5 +1,9 @@
 {
   "exclude": ["**/node_modules", "**/.*/"],
-  "include": ["**/*.ts", "**/*.tsx"],
+  "include": [
+    "**/*.ts",
+    "**/*.tsx",
+    "../magento-search/components/SearchForm/ProductFiltersProSearchField.tsx",
```
Comment (Giovanni-Schroevers) [inline packages/magento-product/tsconfig.json:6]: It seems like we are trying to add a file from outside of this package to this file which I assume is not possible?

File: examples/magento-graphcms/components/Layout/LayoutNavigation.tsx [hunk 3]
```diff
@@ -145,16 +145,7 @@ export function LayoutNavigation(props: LayoutNavigationProps) {
...
             <DesktopNavActions>
-              <ProductFiltersProSearchField
-                sx={
-                  {
-                    // display: { xs: 'none', md: 'block' },
-                    // width: '400px',
-                  }
-                }
-                fullWidth
-                input={{ placeholder: i18n._(/* i18n*/ `Search all products...`) }}
```
Comment (Giovanni-Schroevers) [inline examples/magento-graphcms/components/Layout/LayoutNavigation.tsx]: I think `input` was accidentally removed here?

**Comment by Giovanni-Schroevers** [comment]
Do we also want to show the skeleton for the `As low as` (above product title)? https://graphcommerce-4d6e1wx93-graphcommerce.vercel.app/en-gb/p/black-magic-gc-259-sock

**Comment by Giovanni-Schroevers** [comment]
On this url the colors lose their count for some reason https://magento-247-git-feature-customer-specific-pricing-graphcommerce.vercel.app/en-gb/search/black/q/dominant_color/1755/colors/1779
Screenshot 2024-07-08 at 17 13 34

**Pull request merged**